CircleciScripts/create_xcframeworks.py

```diff
@@ -0,0 +1,108 @@
+import os
+import sys
+
+from framework_list import frameworks
+from functions import log, run_command
+
+EXCLUDE_FROM_XCFramework = [
+    # This isn't a real framework
+    "AWSiOSSDKv2",
+    # Legacy frameworks not built or packaged
+    "AWSAuth",
+    # AWSMobileClient is named as AWSMobileClientXCF and will be added later.
+    "AWSMobileClient",
+]
+
+IOS_DEVICE_ARCHIVE_PATH = "./output/iOS/"
+IOS_SIMULATOR_ARCHIVE_PATH = "./output/Simulator/"
+XCFRAMEWORK_PATH = "./output/XCF/"
+
+def is_framework_included(framework):
+    return framework not in EXCLUDE_FROM_XCFramework
+
+def create_archive(framework, projectFile, isDevice):
```

Comment:
here and throughout: our python style is to use `snake_case` rather than `camelCase`.

Reply:
Updated